common/lib/capa/capa/inputtypes.py

```diff
@@ -85,6 +85,7 @@ def __init__(self, status, gettext_func=unicode):
         names = {
             'correct': _('correct'),
             'incorrect': _('incorrect'),
+            'partially-correct': _('partially-correct'),
```

Comment:
`git blame` may help you, as well as inspecting the edit history of the
file. The other way is to trace through and figure out how these strings
are used. Do you know what this is doing? If not it doesn't seem right to
just blindly add a new entry without understanding what the code is doing.

On Fri, Jun 26, 2015 at 9:25 AM, Colin-Fredericks <notifications@github.com>
wrote:

> In common/lib/capa/capa/inputtypes.py
> <https://github.com/edx/edx-platform/pull/8574#discussion_r33354950>:
>
> > @@ -85,6 +85,7 @@ def __init__(self, status, gettext_func=unicode):
> >          names = {
> >              'correct': _('correct'),
> >              'incorrect': _('incorrect'),
> > +            'partially-correct': _('partially-correct'),
>
> Ok. Who programmed this part originally, so I can ask them what's going on
> here and whether it's user-facing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/8574/files#r33354950>.
>